src/components/Expandable/ExpandableSlots.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  import Portal from '../Portal.svelte';
+  import { expandAnimation } from './helpers';
+
+  let parentRef = null;
+  let isExpanded = false;
+  let isFullVisible = false;
+
+  function showFullScreenComponent() {
+    console.log('-->', 1);
+    isExpanded = true;
+  }
+
+  function hideFullScreenComponent() {
+    isFullVisible = false;
+    setTimeout(() => (isExpanded = false), 50);
+  }
+
+  function endOpening() {
+    // calls before block is fullScreen, need small timeout
```

Comment:
why? )

Reply:
Because calculations start before block is fullScreen and we can see interface hanging. I should white it in the comment)

Reply:
exactly )